db/migrate/20231110010253_workflow_execution.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Migration to add WorkflowExecution table
+class WorkflowExecution < ActiveRecord::Migration[7.1]
+  def change # rubocop:disable Metrics/MethodLength
+    create_table :workflow_executions do |t|
+      t.jsonb :metadata, null: false, default: WorkflowMetadata.dump(
+        WorkflowMetadata.new(workflow_name: '', workflow_version: '')
+      )
+      t.jsonb :workflow_params
+      t.string :workflow_type
+      t.string :workflow_type_version
+      t.string :tags, array: true
+      t.string :workflow_engine
+      t.string :workflow_engine_version
+      t.jsonb :workflow_engine_parameters
+      t.string :workflow_url
+      t.string :run_id
+      t.references :submitter
+      # states to be defined in a later migration
+      # t.enum :state, enum_type: :workflow_execution_state, default: 'tbd'
```

Comment:
Can we just add `state` as a string for now, later we can change this type as we define the different possible values.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/308/commits/b93ceec0d2aa3386add4aafbf34c30d56610ee7f